src/MinimumNodeService.cpp

```diff
@@ -57,11 +57,120 @@ void MinimumNodeService::setController(Controller *cntrl)
 // * SQU
 // * NNRST
 // * NNRSM
+
```

Comment:
The comments above here relate to the method handleMessage(). Please move these comments down to just before this method.